app/generator.js

```diff
@@ -49,11 +49,16 @@ export default class Generator extends Base {
           }], function (answers) {
             this.skipConfig = answers.skipConfig;
 
-            this.filters = this._.defaults(this.config.get('filters'), {
-              bootstrap: true,
-              uibootstrap: true,
-              jasmine: true
-            });
+            if (this.skipConfig) {
+              this.filters = this._.defaults(this.config.get('filters'), {
+                bootstrap: true,
+                uibootstrap: true,
+                jasmine: true
```

Comment:
@Awk34 I'm wondering if we even need to be setting the defaults here. I assume they are for migration purposes, can you think of any other reasons, use cases?